pkg/certificates/certificates.go

```diff
@@ -21,8 +23,25 @@ func RunCertificatesScript(namespace, logStoreName, workDir, script string) (err
 	log.V(3).Info("Cert generation", "out", result, "err", err)
 	if result != "" {
 		updated = true
-		log.Info("cert_generation output", "output", result)
+		dumpLogs(result)
 	}
 	log.V(3).Info("Returning", "err", err, "updated", updated)
 	return err, updated
 }
+
+func dumpLogs(raw string) {
+	genLogs := []map[string]string{}
+	err := yaml.Unmarshal([]byte(raw), &genLogs)
+	if err == nil {
+		for _, eventlog := range genLogs {
+			keyvalues := []interface{}{}
+			for k, v := range eventlog {
+				keyvalues = append(keyvalues, k, v)
+			}
+			log.Info("cert_generation output", keyvalues...)
+		}
+		return
+	}
+	log.Error(err, "Unable to unmarshal cert_generation to structured output")
+	log.Info("cert_generation output", "output", raw)
```

Comment:
you have both an error and info level here? why not move the `"output", raw` into the Error log?

Reply:
This is intentional.  The PR adds the ability to have structured logs from the script which should help clarify why cert regen is happening.  What i wanted here was to preserve the previous output in the event we could not parse the output to provide some indicator that regen occurred.  So the logs here are:  Error and tell me why; give me the original output too.